demos/functions/views/demo/workbox-background-sync/sw.hbs

```diff
@@ -0,0 +1,16 @@
+{{{WORKBOX_SW_IMPORT}}}
+
+const bgSyncPlugin = new workbox.backgroundSync.Plugin('myQueueName');
+
+const networkOnlyStrategy = new workbox.strategies.NetworkOnly({
+  plugins: [bgSyncPlugin],
+});
+
+workbox.routing.registerRoute(new workbox.routing.Route(
+  ({url}) => url.pathname === '/example.txt',
```

Comment:
This should be relative as well I think.

Reply:
pathname gives the full absolute path hence i've used it. Also wasn't sure about `endsWith` support.